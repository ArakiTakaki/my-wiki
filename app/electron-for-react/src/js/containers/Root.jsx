import React from 'react';
import PropTypes from 'prop-types';
import Index from '../pages';
import Store from '../store';
import { Provider } from 'mobx-react';
import DevTools from 'mobx-react-devtools';
const store = {
  store: new Store()
};
class Root extends React.Component {
  render() {
    return (
      <div>
        <Provider {...store}>
          <Index />
        </Provider>
        {process.env.NODE_ENV === 'development' ? <DevTools /> : null}
      </div>
    );
  }
}

Root.propTypes = {
  children: PropTypes.any
};

export default Root;
