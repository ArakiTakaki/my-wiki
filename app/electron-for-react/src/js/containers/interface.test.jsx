import React from 'react';
import renderer from 'react-test-renderer';

import Root from './Root';

test('interface Snapshot', () => {
  const component = renderer.create(<Root />).toJSON();
  expect(component).toMatchSnapshot();
});
