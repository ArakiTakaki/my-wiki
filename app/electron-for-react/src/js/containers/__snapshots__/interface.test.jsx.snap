// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`interface Snapshot 1`] = `
<div>
  <div>
    <p>
       
      
       
    </p>
  </div>
</div>
`;
